pdc/apps/tree/models.py

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright (c) 2015 Red Hat
+# Licensed under The MIT License (MIT)
+# http://opensource.org/licenses/MIT
+#
+from django.db import models
+from jsonfield import JSONField
```

Comment:
This needs to be reflected in `requirements/production.txt` so that it gets installed when `./setup.py develop`, `./setup.py install` are being run.